### ISSUE TITLE
Annotations are temperamental

### ISSUE THREAD

**Issue by jakzilla** [JoinColony/colonyDapp#2777]
When a motion is created, often the justification does not show.

In the below image I had added justifications for both the motion and the objection. Initially the "motion" side was not showing and the "objection" side was showing. After a refresh, the objection justification disappeared and the motion side appeared.

<img width="1321" alt="Screenshot 2021-10-07 at 11 58 21" src="https://user-images.githubusercontent.com/4774274/136371467-e0b67c98-3517-4cf1-ae7e-61fbbb2d8be5.png">

**Comment by agustif** [comment]
Hey @rdig, so this is related somewhat to #2654 right?

**Comment by rdig** [comment]
> Hey @rdig, so this is related somewhat to #2654 right?

No, #2654 is about uploading to IPFS, this issue is about fetching said data.

For fetching, pinata is our backup, our main one is Cloudflare, and that never goes down.

However, the problem is not the gateway itself, but the actual data that lives on IPFS, both Cloudflare and Pinata facilitate access to that data, but they do not provide explicitly
